engine/src/main/java/org/terasology/entitySystem/prefab/internal/PojoPrefabManager.java

```diff
@@ -51,7 +60,8 @@ public Prefab getPrefab(String name) {
      */
     @Override
     public boolean exists(String name) {
-        return Assets.getPrefab(name) != null;
+        Preconditions.checkArgument(!Strings.isNullOrEmpty(name));
```

Comment:
I would argue it is ok if name is empty, so the contract can be loosened a little.